meson.build

```diff
@@ -181,14 +183,15 @@ endif
     config.set('WORDS_BIGENDIAN', 1)
   endif
 
+  # Execute subdir to create config.h for this pass
+  subdir(entry.get('header-subdir'))
```

Comment:
Could you please add a comment that subdir() relies on the `config` variable being set in the loop? It's not quite obvious otherwise and a future maintainer might miss the context.

Reply:
Roger.